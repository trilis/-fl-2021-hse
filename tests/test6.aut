ALPH_SIZE=2
START_ST=1
ST_NUM=3 
ED_NUM=4
"0" "1" "2"
STATE 1, term 
STATE 2, term
STATE 3, notterm
EDGE 1, 2, "0"
EDGE 2, 1, "1"
EDGE 1, 2, "1"
EDGE 2, 3, "0"
