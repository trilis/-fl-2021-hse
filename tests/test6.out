Exception: invalid alphabet symbol in transition